boot.php

```diff
@@ -854,11 +854,13 @@ function check_addons(App $a)
 	return;
 }
 
-function get_guid($size = 16, $prefix = "")
+function get_guid($size = 16, $prefix = '')
 {
-	if ($prefix == "") {
+	if (is_bool($prefix)) {
```

Comment:
If `$prefix` is `true`, this function should generate a default prefix as well.

Reply:
Done.